packages/sdk/src/Base.ts

```diff
@@ -549,12 +549,22 @@ class Base {
 
   getSupportedAssets () {
     const supported : any = {}
+
+    const unsupportedAssets: any = {
```

Comment:
i think ideally this mapping is constructed dynamically based on the config tokens

Reply:
you're right, this was an initial approach i took, but is not necessary. will revert. thanks for the catch!